test/jdk/java/awt/Mouse/GetMousePositionTest/GetMousePositionWithPopup.java

```diff
@@ -90,24 +110,28 @@ public void mouseMoved(MouseEvent e) {
                 frame2 = new Frame();
                 frame2.setBounds(120, 120, 120, 120);
                 frame2.setVisible(true);
-                frame2.addMouseMotionListener(new MouseMotionAdapter() {
-                    @Override
-                    public void mouseMoved(MouseEvent e)
-                    {
-                        Point positionInFrame2 = frame2.getMousePosition();
-                        int deltaX = Math.abs(50 - positionInFrame2.x);
-                        int deltaY = Math.abs(50 - positionInFrame2.y);
-                        if (deltaX > 2 || deltaY > 2) {
-                            throw new RuntimeException("Wrong position reported. Should be [50, 50] but was [" +
-                                    positionInFrame2.x + ", " + positionInFrame2.y + "]");
-                        }
+            }
```

Comment:
So are you confident the requested location will be honoured ?
I seem to remember a test framework fix in which you had to work quite hard to be sure of the real location.
And the implied "170-120=50" might be clearer if we had variables not just literal numbers ?

Reply:
@prrace I agree, adding PassFailJFrame's [syncLocationToWindowManager()](https://github.com/openjdk/jdk/blob/a8871f5d26e5cb42c031c7b736ec30b1b147a2bc/test/jdk/java/awt/regtesthelpers/PassFailJFrame.java#L343) to this test will make it more robust.
I'll change the literals to variables to make the context clear.

Reply:
Test updated.